ods_ci/tests/Resources/Common.robot

```diff
@@ -342,17 +342,29 @@ Run And Watch Command
   ...    Output Should Contain: Verify an excpected text to exists in command output.
   ...    Output Should Not Contain: Verify an excpected text to not exists in command output.
   [Arguments]    ${command}    ${timeout}=10 min   ${output_should_contain}=${NONE}    ${output_should_not_contain}=${NONE}
+  ...            ${cwd}=${NONE}
   Log    Watching command output: ${command}   console=True
+  ${is_test}=    Run keyword And Return Status    Variable Should Exist     ${TEST NAME}
+  IF    ${is_test} == ${FALSE}
+    ${incremental}=    Generate Random String    5    [NUMBERS]
+    ${TEST NAME}=    Set Variable    testlogs-${incremental}    
+  END
   ${process_log} =    Set Variable    ${OUTPUT DIR}/${TEST NAME}.log
   ${temp_log} =    Set Variable    ${TEMPDIR}/${TEST NAME}.log
-  Set Test Variable    ${process_log}
-  Set Test Variable    ${temp_log}
+  Set Variable    ${process_log}
+  Set Variable    ${temp_log}
```

Comment:
I don't think you need to call "Set Variable", since it is already called in lines 352-353.

Reply:
true, I can just remove the lines. thanks